Kernel/TTY/TTY.h

```diff
@@ -40,14 +41,28 @@ class TTY : public CharacterDevice {
 
     TTY(unsigned major, unsigned minor);
     void emit(u8);
+    virtual void echo(u8);
+
+    bool can_do_backspace();
```

Comment:
This function does not mutate state so it should be const:
```suggestion
    bool can_do_backspace() const;
```